src/components/profileConf/profileConfig/index.js

```diff
@@ -2,23 +2,16 @@ import React, { useState } from "react";
 import { ProfileConf } from '../index.js';
```

Comment:
The code seems well, but the nomenclature to declare components in React is **PascalCase**, so the right way to name the components would be: ChangingUser (Let's try to be the more descriptive as we can), ChangingPass, ProfileConf, ProfileConfig just like the function components.

Also, I saw there is a component inside another component in the folders just like this: _profileConf/profileConfig/index.js, changingPass, changingUsr_. 

Every component should be separated, or if ChangingUser and ChangingPass are components that only will be rendered in a specific part of the application you can use the folder "Containers" to store them just like this: _/containers/ProfileConfig/index.js_ <- over here you would import the components, this way the project is more readable when we need to debug or refactorize.

Besides, I didn't view the functionality of the first folder "profileConf" which is wrapping the "profileConfig" folder.

I won't request changes because the imports will be affected, but I advise using the current file composition because it's cleaner. 😉 

This is the doc about the recommendations to make easier the development with React 😄 
https://es.reactjs.org/docs/faq-structure.html#grouping-by-features-or-routes